webpack.config.js

```diff
@@ -20,19 +20,16 @@ module.exports = [{
         // Needed by Cesium for multiline strings
         sourcePrefix: ''
     },
-    amd: {
-        // Enable webpack-friendly use of require in cesium
-        toUrlUndefined: true
-    },
     node: {
         // Resolve node module use of fs
-        fs: "empty"
+        fs: "empty",
+        Buffer: false,
+        http: "empty",
+        https: "empty",
+        zlib: "empty"
     },
     resolve: {
-        alias: {
-            // Cesium module name
-            cesium: path.resolve(__dirname, cesiumSource)
-        }
+        mainFields: ['module', 'main']
```

Comment:
Why is this needed?

Reply:
This is what tell's webpack to use node-based module resolution, similar to how rollup node resolve works in other projects you're familiar with.  From what I know this is now the "standard" way people use front-end libraries from nom.